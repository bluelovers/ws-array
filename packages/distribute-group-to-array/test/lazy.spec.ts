import distributeGroupToArray from '../src/index';

test(`distributeGroupToArray`, () =>
{
	let data = {
		"18056a75": [
			{
				"name": "Forest",
			},
			{
				"name": "Forest",
			},
			{
				"name": "Forest",
			},
		],
		"57a555ae": [
			{
				"name": "Plains",
			},
			{
				"name": "Plains",
			},
			{
				"name": "Plains",
			},
		],
	};

	let actual = distributeGroupToArray(data);

	expect(actual).toMatchSnapshot();

});
