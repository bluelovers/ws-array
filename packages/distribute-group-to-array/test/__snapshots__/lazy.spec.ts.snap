// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`distributeGroupToArray 1`] = `
Array [
  Object {
    "name": "Forest",
  },
  Object {
    "name": "Plains",
  },
  Object {
    "name": "Forest",
  },
  Object {
    "name": "Plains",
  },
  Object {
    "name": "Forest",
  },
  Object {
    "name": "Plains",
  },
]
`;
